oval_graph/converter.py

```diff
@@ -51,7 +63,7 @@ def get_tag(self):
             return str(self.tree.tag)
         return ""
 
-    def to_JsTree_dict(self):
+    def to_JsTree_dict(self, option=False):
```

Comment:
I think `option` isn't a good variable name, because it doesn't specify which option, what the option does. Try to name your variables in a self-explanatory way. For example `hide_passing_tests` could be a good name, as it's obvious what happens when you set the variable to true.